### DISCUSSION TITLE
Parameter for Verbose Folder Names

### DISCUSSION THREAD

**Discussion by N64Core** [Emersont1/itchio#56] [Ideas]
I'm unsure how feasible this is, but in my use case I would like the folder names to be easily searchable using the display title of the game in question e.g. "Lancer Core Book First Edition PDF" vs "corebook-pdf" because certain publishers tend to use fairly generic names for their titles, it can make finding them later a bit of a memory game. Based on my understanding it may not be too difficult, since this data is contained on the primary landing page in the HTML title tag: 

> <title>Lancer Core Book: First Edition PDF by Massif Press</title>

From there you would only need to check for the position of the first instance of ' by ' from the end of the string, delete everything afterwards, and sanitize the folder name to be in line with your filesystem's naming conventions (in this case stripping the colon from the title to render it safe for NTFS formatted disks).

**Comment by N64Core** [comment]
I created a fork for seeing how feasible this is exactly: [Link](https://github.com/N64Core/itchio) 

I haven't figured out exactly how to run it for testing purposes quite yet but it seems like a simple enough addition, most of the functionality was already present but just needed an argument for switching between modes

**Reply by Emersont1** [reply]
so to test it in a venv you just need to run `pip install git+https://https://github.com/N64Core/itchio@main`

I'd rather avoid using global variables for config, but this may mean i need to add a conf object to be passed around
